roscala/src/main/scala/coop/rchain/rosette/Main.scala

```diff
@@ -15,11 +15,8 @@ object Main extends App {
 
   Parser.parse(bytes) match {
     case Right(opCodes) =>
```

Comment:
Not sure what you mean with unified scalafmt configuration?

Reply:
@tymm The chnge here is really `Location.PLACEHOLDER` -> `Limbo` right?

But because of different formatting, the diff is bigger (whole `val iniState = ...`). This is not an big of an issue here but might be when reviewing bigger project. 
`comm` has formatter (scalafmt + config file) that unifies alignment and all that stuff among different IDEs, editors and developers. See https://github.com/rchain/rchain/blob/dev/comm/build.sbt#L15 and https://github.com/rchain/rchain/blob/dev/comm/.scalafmt.conf for detail.

Reply:
Above is not particular comment to this PR, more general note - issue is visible here.

Reply:
@tymm I'm sorry, I'm dumb. You were reviewing my `scalafmt` PR, so you know what I mean :)

Reply:
@rabbitonweb Yeah, I thought that you meant that but wasn't 100% sure. We'll totally steal the `scalafmt` config file. Makes sense. Thanks!